Sources/SwiftDocC/Checker/Checkers/NonInclusiveLanguageChecker.swift

```diff
@@ -161,31 +163,33 @@ public struct NonInclusiveLanguageChecker: Checker {
 }
 
 /// The default list of terms to look for in documentation.
-fileprivate let builtinExcludedTerms: [NonInclusiveLanguageChecker.Term] = [
+private let builtinExcludedTerms: [NonInclusiveLanguageChecker.Term] = [
     NonInclusiveLanguageChecker.Term(
         expression: #"black\W*list\w{0,2}"#,
         message: "Choose a more inclusive alternative that’s appropriate to the context, such as deny list/allow list or unapproved list/approved list.",
         replacement: "deny list"
     ),
     NonInclusiveLanguageChecker.Term(
         expression: #"master\w{0,2}"#,
-        message: #"Don't use "master" to describe the relationship between two devices, processes, or other things. Use an alternative that's appropriate to the context, such as "main" and "secondary" or "host" and "client"."#,
+        message:
+            #"Don't use "master" to describe the relationship between two devices, processes, or other things. Use an alternative that's appropriate to the context, such as "main" and "secondary" or "host" and "client"."#,
         replacement: "primary"
     ),
     NonInclusiveLanguageChecker.Term(
         expression: #"slave\w{0,2}"#,
-        message: #"Don't use "slave" to describe the relationship between two devices, processes, or other things. Use an alternative that's appropriate to the context, such as "main" and "secondary" or "host" and "client"."#,
+        message:
+            #"Don't use "slave" to describe the relationship between two devices, processes, or other things. Use an alternative that's appropriate to the context, such as "main" and "secondary" or "host" and "client"."#,
         replacement: "secondary"
     ),
     NonInclusiveLanguageChecker.Term(
         expression: #"white\W*list\w{0,2}"#,
         message: "Choose a more inclusive alternative that’s appropriate to the context, such as deny list/allow list or unapproved list/approved list.",
         replacement: "allow list"
-    )
+    ),
 ]
 
 /// The regular expressions for the default term list.
-fileprivate let defaultRegularExpressions: [String: NSRegularExpression] = builtinExcludedTerms.reduce(into: [:]) { result, term in
+private let defaultRegularExpressions: [String: NSRegularExpression] = builtinExcludedTerms.reduce(into: [:]) { result, term in
```

Comment:
Are fileprivate and private equivalent?

Reply:
For top-level declarations they are. For the first few releases (maybe until Swift 3.x or so? 🤔) fileprivate didn't exist.